### DISCUSSION TITLE
External SD card access

### DISCUSSION THREAD

**Discussion by craigyjp** [probonopd/MiniDexed#427] [Show and tell]
Finally managed to get some SD to micro SD extension cables from Australia today, very helpful company actually sent me 3 for good luck when I only ordered 2 and they arrived very quickly considering their journey and the situation with air freight etc to the Ukraine. So now my MiniDexed build is complete and the case closed and I can apply updates without undoing 30 or more screws.

![20230127_182348.jpg](https://user-images.githubusercontent.com/51026305/215140480-8ab884a6-ee5a-4d10-ae62-376b761b3889.jpg)

**Comment by Banana71** [comment]
Awesome, is that the back of your CME UF70?
Looks very professional 😍

**Comment by craigyjp** [comment]
Yeah it's the back of the CME, they have an expansion port at the back for
add on synth modules, although not easy to find anymore it does provide an
internal MIDI and power to drive anything you build in there.

**Reply by Banana71** [reply]
How did you do the caption?

**Reply by craigyjp** [reply]
Do you mean the text? It's just printed onto glossy photo paper and laminated with a matte lamination
